vite.config.mts

```diff
@@ -84,6 +84,11 @@ export default {
       },
     }),
   ],
+  resolve: {
+    alias: {
+      "@careConfig": path.resolve(__dirname, "./care.config.ts"),
+    },
```

Comment:
```suggestion
    alias: {
      "@careConfig": path.resolve(__dirname, "./care.config.ts"),
      "@": path.resolve(__dirname, "/src")
    },
```

This should fix https://github.com/ohcnetwork/care_fe/issues/6050

Reply:
Just confirming, we decided on `@/` right?

Reply:
@rithviknishad @khavinshankar the usages will also have to be replaced. lets not do it in this PR

Yes we decided for `@/`